src/applications/simple-forms/mock-simple-forms-patterns/pages/mockArrayMultiPageBuilderItemPage1.js

```diff
@@ -1,12 +1,24 @@
+import React from 'react';
 import {
   addressNoMilitarySchema,
   addressNoMilitaryUI,
   titleUI,
 } from 'platform/forms-system/src/js/web-component-patterns';
 import { VaTextInputField } from 'platform/forms-system/src/js/web-component-fields';
+import ArrayBuilderCancelAddingButton from '../arrayBuilder/components/ArrayBuilderCancelAddingButton';
 
 /** @type {PageSchema} */
 export default {
+  ContentBeforeButtons: ({ router }) => (
```

Comment:
these options might be able to be reduced to `noun` and `pluralNoun` if we want to